_data/simple-icons.json

```diff
@@ -3442,6 +3442,11 @@
             "hex": "79BD42",
             "source": "https://comicfury.com/images/gator-icon-black.png"
         },
+        {
+            "title": "comma",
+            "hex": "51FF00",
+            "source": "https://github.com/commaai/connect/blob/d431d977fb79e01a0f565f259a98ec3819b2b3c5/public/icon-vector.svg"
```

Comment:
Ooh actually I misinterpreted this URL. This isn't an outreach/community repository, it's a specific subproduct. We should use the logo in their footer, https://comma.ai/images/logo-icon-shape.svg. Can you update the source here and verify it still matches with your icon here?

Reply:
[de23288](https://github.com/simple-icons/simple-icons/pull/11034/commits/de232882dd361e50dd57c98a1f156dd3dfac5a1c)